test/tinycore.txt

```diff
@@ -2,8 +2,15 @@ These are binaries from
     http://tinycorelinux.net
 with the following patch applied:
 
+Source is available http://www.tinycorelinux.net/6.x/x86/release/src/
+
+curl -O http://www.tinycorelinux.net/6.x/x86/release/distribution_files/vmlinuz64
+# c4f46b5cd826d7a765a3e85439309cfc  vmlinuz64
+mv vmlinuz64 vmlinuz
+curl -O http://www.tinycorelinux.net/6.x/x86/release/distribution_files/core.gz
+mv core.gz initrd.gz
 mkdir initrd
-( cd initrd ; zcat ../initrd.gz | sudo cpio -idm )
-sudo sed -i '/^# ttyS0$/s#^..##' initrd/etc/securetty 
-sudo sed -i '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
+( cd initrd ; gzcat ../initrd.gz | sudo cpio -idm )
+sudo sed -i -e '/^# ttyS0$/s#^..##' initrd/etc/securetty 
+sudo sed -i -e '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
 ( cd initrd ; find | sudo cpio -o -H newc ) | gzip -c > initrd.gz
```

Comment:
Yeah sorry I missed updating that - I used `find .` (if you do `../` you'll have initrd/ in the path so the boot will probably not find init

The intent for the md5sum was just to sanity check the vmlinuz I linked to was in fact the vmlinuz in git

I can easily refactor this PR into a script for both this and for userboot.so for FreeBSD. I want to potentially look at other OS support or testing boots so this might make sense.